test/unit/sql/select.test.js

```diff
@@ -706,11 +715,12 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
               .replace(/\]/g, Support.sequelize.dialect.TICK_CHAR_RIGHT),
           ],
         }), {
-          default: 'SELECT \'* FROM [User]; DELETE FROM [User];SELECT [id]\' FROM [User];',
-          ibmi: 'SELECT \'* FROM "User"; DELETE FROM "User";SELECT "id"\' FROM "User"',
-          db2: 'SELECT \'* FROM "User"; DELETE FROM "User";SELECT "id"\' FROM "User";',
-          snowflake: 'SELECT \'* FROM "User"; DELETE FROM "User";SELECT "id"\' FROM "User";',
-          mssql: 'SELECT [* FROM User; DELETE FROM User;SELECT id] FROM [User];',
+          // TODO: the attribute should be escaped as an identifier, not a string
```

Comment:
while this is inconsistent and needs to be fixed, I'm not going to do it in this PR, it's big enough and unrelated